Add Solweig gpu

nvnsudharsan opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with a PR, please let the
right people know. There are language-specific teams for reviewing recipes.

| Language        | Name of review team           |
| --------------- | ----------------------------- |
| python          | `@conda-forge/help-python`    |
| python/c hybrid | `@conda-forge/help-python-c`  |
| r               | `@conda-forge/help-r`         |
| java            | `@conda-forge/help-java`      |
| nodejs          | `@conda-forge/help-nodejs`    |
| c/c++           | `@conda-forge/help-c-cpp`     |
| perl            | `@conda-forge/help-perl`      |
| Julia           | `@conda-forge/help-julia`     |
| ruby            | `@conda-forge/help-ruby`      |
| Rust            | `@conda-forge/help-rust`      |
| Go              | `@conda-forge/help-go`        |
| other           | `@conda-forge/staged-recipes` |

Once the PR is ready for review, please mention one of the teams above in a
new comment. i.e. `@conda-forge/help-some-language, ready for review!`
Then, a bot will label the PR as 'review-requested'.

Due to GitHub limitations, first time contributors to conda-forge are unable
to ping conda-forge teams directly, but you can [ask a bot to ping the team][1]
using a special command in a comment on the PR to get the attention of the
`staged-recipes` team. You can also consider asking on our [Zulip chat][2]
if your recipe isn't reviewed promptly.

[1]: https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team
[2]: https://conda-forge.zulipchat.com

All apologies in advance if your recipe PR does not receive prompt attention.
This is a high volume repository and the reviewers are volunteers. Review times
vary depending on the number of reviewers on a given language team and may be days
or weeks. We are always looking for more staged-recipe reviewers. If you are
interested in volunteering, please contact a member of @conda-forge/core.
We'd love to have the help!
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

nvnsudharsan committed:
Create meta.yaml

nvnsudharsan committed:
Update meta.yaml

Commit:
meta.yaml moved to folder

nvnsudharsan committed:
Merge branch 'main' into main

nvnsudharsan committed:
Rename meta.yaml to recipe.yaml

nvnsudharsan committed:
Update recipe.yaml

nvnsudharsan committed:
Rename recipe.yaml to meta.yaml

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/Solweig-GPU/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipes/Solweig-GPU/meta.yaml**:

 * ℹ️ License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

Documentation on acceptable licenses can be found [here]( https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions ).


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/17745108460](https://github.com/conda-forge/conda-forge-webservices/actions/runs/17745108460). Examine the logs at this URL for more detail.</sub>

nvnsudharsan committed:
Update meta.yaml

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/Solweig-GPU/meta.yaml```) and found it was in an excellent condition.

nvnsudharsan commented:
@conda-forge/help-python ready for review!

nvnsudharsan committed:
Update meta.yaml

nvnsudharsan commented:
@conda-forge/staged-recipies ready for review!

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan commented:
> Hi @nvnsudharsan , I'm not a reviewer. I'm a person like you who just submitted a pull request to staged-recipes and I'm very worried about the reviewers not getting to pull requests like yours for 6 months. Do you know why your pull request here was totally ignored?

@damonturney I have no idea

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Update recipes/Solweig-GPU/meta.yaml

Co-authored-by: Travis Hathaway <travis.j.hathaway@gmail.com>

nvnsudharsan committed:
Update recipes/Solweig-GPU/meta.yaml

Co-authored-by: Travis Hathaway <travis.j.hathaway@gmail.com>

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/Solweig-GPU/meta.yaml```) and found some lint.

Here's what I've got...


For **recipes/Solweig-GPU/meta.yaml**:

 * ❌ Non noarch packages should have python requirement without any version constraints.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/27375696447](https://github.com/conda-forge/conda-forge-webservices/actions/runs/27375696447). Examine the logs at this URL for more detail.</sub>

nvnsudharsan committed:
Update version to 1.2.22 and sha256 hash

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Refactor meta.yaml for SOLWEIG-GPU package

Updated the meta.yaml file for SOLWEIG-GPU package to remove Python version constraints and improve the package description formatting.

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/Solweig-GPU/meta.yaml```) and found it was in an excellent condition.

nvnsudharsan committed:
Update sha256 checksum in meta.yaml

nvnsudharsan committed:
Refactor meta.yaml for Python compatibility and build

Updated Python version requirements and removed skip condition for builds.

nvnsudharsan committed:
Update entry points in meta.yaml for Solweig-GPU

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu

nvnsudharsan committed:
Merge branch 'main' into solweig_gpu
